README.md

```diff
@@ -1 +1,23 @@
 # hw01
+
+## This is my brief Biography!
+
+My name is Eleanor Anderson. I am a current fourth year undergraduate student majoring in Economics and Psychology. I am planning on graduating this quarter and am applying to graduate schools. I wanted to take this class becuase I have had a handful of classes that use R but I never truly learned how R works, I mostly learned how to plug and chug. I am **very** *excited* to learn how to actually use R!
```

Comment:
Hi Eleanor! Thank you for sharing personal details about your hobbies and interests! Your short biography was easy to follow! My only comment is in the future, when you write larger texts try to use bold or italic with the same purpose, for example, using just bold to highlight important details of your bio or italic just for your majors. I think it is a great way to grab the reader’s attention.